get_github_coderabbit_review/coderabbit_fetcher/cache/file_cache.py

```
@@ -0,0 +1,501 @@
+
+            if namespace_dir.exists():
+                try:
+                    # Count files before removal
+                    removed_count += sum(
+                        1
+                        for root, dirs, files in os.walk(namespace_dir)
+                        for f in files
+                        if f.endswith(".cache")
+                    )
+                    shutil.rmtree(namespace_dir)
+                except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)